### PR TITLE
[wix-style-react] - <AddressInput/> - add highlighting

### PR DESCRIPTION
**Pull request by gendeld**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6594

File: packages/wix-style-react/src/AddressInput/test/AddressInput.visual.js [hunk 1]
```diff
@@ -1,30 +1,121 @@
...
+        closed: true,
+      },
     ],
   },
 ];
 
+class InteractiveEyeTest extends React.Component {
+  async componentDidMount() {
+    const { disabled, closed } = this.props;
+    if (disabled || closed) return;
```
Comment (TalZa1810) [inline packages/wix-style-react/src/AddressInput/test/AddressInput.visual.js:92]:
```suggestion
    if (disabled || closed){
        return;
     }
```

File: packages/wix-style-react/src/AddressInput/test/AddressInput.visual.js [hunk 2]
```diff
@@ -1,30 +1,121 @@
...
-  its.forEach(({ it, props }) => {
-    storiesOf(
-      `${AddressInput.displayName}${describe ? '/' + describe : ''}`,
-      module,
-    ).add(it, () => <AddressInput {...commonProps} {...props} />);
+  its.forEach(({ it, props, closed }) => {
+    storiesOf(`AddressInput${describe ? '/' + describe : ''}`, module).add(
+      it,
+      () => (
+        <Box direction="vertical">
```
Comment (TalZa1810) [inline packages/wix-style-react/src/AddressInput/test/AddressInput.visual.js:113]: why do we need so many Box wrappers?
3 Box wrappers (2 here and one within `<InteractiveEyeTest/>`

**Review by TalZa1810 — APPROVED** [review]
seems good. few tiny comments.

**Pull request merged**